articles/building-a-python-package-and-publishing-on-pypi/index.md

```diff
@@ -0,0 +1,169 @@
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+```
+
+Now on to a very important part of building our package, which is the *setup.py* file, this is the tool that publishes our package to PyPI and is used to get all the information it needs.
+
+Before we proceed let’s rearrange for packaging, create a new folder with any name of your choice and put the *init.py* file in the new folder you just created. Then create a file and save it as *setup.py*, this file will hold all the important information the compiler will need to set up our package.
+
+Then add the following code below:
+
+```python
+import pathlib
+from setuptools import setup, find_packages
+
+HERE = pathlib.Path(__file__).parent
+
+VERSION = '0.0.0.1'
+PACKAGE_NAME = 'findlcm'
+AUTHOR = 'james'
+AUTHOR_EMAIL = 'james@sandy.com'
+URL = 'https://github.com/jamessandy/lcmfinder'
+
+LICENSE = 'MIT'
+DESCRIPTION = 'A package that helps find Lowest common multiple of a number'
+LONG_DESCRIPTION = (HERE / "README.md").read_text()
+LONG_DESC_TYPE = "text/markdown"
+
+INSTALL_REQUIRES = [
+      'numpy',
+      'pandas'
+]
+
+setup(name=PACKAGE_NAME,
+      version=VERSION,
+      description=DESCRIPTION,
+      long_description=LONG_DESCRIPTION,
+      long_description_content_type=LONG_DESC_TYPE,
+      author=AUTHOR,
+      license=LICENSE,
+      author_email=AUTHOR_EMAIL,
+      url=URL,
+      install_requires=INSTALL_REQUIRES,
+      packages=find_packages()
+      )
+
+```
+
+The first chunk of code imports all the tools we need to set up our library, the *VERSION* will be the release number for the package, the *PACKAGE NAME* will be a unique name you want to call your package, the *AUTHOR* and *AUTHOR EMAIL* will be your details and finally, the URL can be a site if you have one or maybe your GitHub Repo.
+
+If your package requires other packages to work then you’ll need to indicate them, for example, if you use other packages to write your code you should add them in the *INSTALL_REQUIRES* because your package depends on them (otherwise it will not work) but this package we just created doesn’t require any and I intentionally put wrote this note, in case you’re trying it out with your package idea.
```

Comment:
Rephrase the last sentence. Just mention that you have given an example for how to handle dependencies. No need to use the word intentionally.